src/Sprite/OP2BmpLoader.cpp

```diff
@@ -8,6 +8,9 @@
 OP2BmpLoader::OP2BmpLoader(std::string bmpFilename, std::string artFilename) :
 	bmpReader(bmpFilename), artFile(ArtFile::Read(artFilename)) { }
 
+OP2BmpLoader::OP2BmpLoader(std::string bmpFilename, Stream::BidirectionalReader& artFileStream) :
```

Comment:
Wasn't sure about adding a comment to the constructor indicating it was useful to allow initializing OP2BmpLoader without first extracting ArtFile from a volume.

Reply:
overcome by other changes